libraries/chain/transaction_context.cpp

```diff
@@ -330,10 +328,6 @@ namespace eosio { namespace chain {
       for( uint32_t i = 1; i <= num_original_actions_to_execute; ++i ) {
          execute_action( i, 0 );
       }
-
-      if( delay != fc::microseconds() ) {
```

Comment:
You can also remove the `delay` member variable from `transaction_context`.

Reply:
Removing `delay` breaks a number of seemingly unrelated tests. Will look into it further.

Reply:
It was my mistake. Done.